src/components/common/layout.js

```diff
@@ -35,6 +35,36 @@ const globalStyles = css`
     font-family: Montserrat, Helvetica, Arial;
     font-size: 100%;
     padding-top: var(--body-skip-height);
+
+    .early-years {
+      --highlight-color: var(--aqua);
+    }
+
+    .school-years {
+      --highlight-color: var(--yellow);
+    }
+
+    .further-education {
+      --highlight-color: var(--midnight);
+
+      .pill-list {
+        a.current-page {
+          color: var(--white);
+        }
+
+        li a:hover {
+          color: var(--white);
+        }
+      }
```

Comment:
I'm okay with this override for now, but this is kinda what I meant that `--white`/`--midnight` is a weird way of naming variables because it means they're not variable. If we had `--highlight-contrast-color` for this purpose, we'd be able to set the `color/contrast-color` pair for each of these classes.

Similarly, `--base-background` might be set to our `--white` and `--base-foreground` as `--midnight` by default.

Reply:
yeah good point, maybe we can have a look at this next week if we have some time left